### PR TITLE
Check activation result

### PR DESCRIPTION
**Pull request by Jamminroot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
This PR will change `cake` build script so that it verifies that license was activated
[AB#69191](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/69191)

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -701,6 +701,20 @@ Task("RegisterUnity").Does(()=>
     // This will produce an error, but that's okay because the project "noproject" is used so that the
     // root isn't opened by unity, which could potentially remove important .meta files.
     ExecuteUnityCommand($"-serial {serialNumber} -username {username} -password {password}", "noproject");
+
```
Comment (Jamminroot) [inline build.cake]:
```suggestion
```

File: build.cake [hunk 2]
```diff
@@ -701,6 +701,18 @@ Task("RegisterUnity").Does(()=>
     // This will produce an error, but that's okay because the project "noproject" is used so that the
     // root isn't opened by unity, which could potentially remove important .meta files.
     ExecuteUnityCommand($"-serial {serialNumber} -username {username} -password {password}", "noproject");
+    var found = false;
+    if (Statics.Context.IsRunningOnUnix())
+    {
+        found = GetFiles("/Library/Application Support/Unity/*.ulf").Count > 0;
+    } else {
+        var localAppDataUnityPath = System.IO.Path.Combine(EnvironmentVariable("LOCALAPPDATA"), @"VirtualStore\ProgramData\Unity\*.ulf");
+        found = GetFiles(localAppDataUnityPath).Count + GetFiles(@"C:\ProgramData\Unity\*.ulf").Count > 0;
```
Comment (MatkovIvan) [inline build.cake]: What if Unity is installed not on `C:`?

Comment (Jamminroot) [reply build.cake]: fixed in next commit

**Review by annakocheshkova — APPROVED** [review]

File: build.cake [hunk 3]
```diff
@@ -701,6 +701,18 @@ Task("RegisterUnity").Does(()=>
     // This will produce an error, but that's okay because the project "noproject" is used so that the
     // root isn't opened by unity, which could potentially remove important .meta files.
     ExecuteUnityCommand($"-serial {serialNumber} -username {username} -password {password}", "noproject");
+    var found = false;
+    if (Statics.Context.IsRunningOnUnix())
```
Comment (guperrot) [inline build.cake:705]: Was this tested locally with local licenses? (on both platforms)

Comment (annakocheshkova) [reply build.cake:705]: Yes

**Pull request merged**